drivers/adc/adc_shell.c

```diff
@@ -32,17 +34,45 @@
 
 #define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
 #include <logging/log.h>
+
 LOG_MODULE_REGISTER(adc_shell);
 
-struct adc_hdl {
-	char *device_name;
-	struct adc_channel_cfg channel_config;
-	uint8_t resolution;
-};
+#define CMD_HELP_ACQ_TIME 			\
+	"Configure acquisition time."		\
+	"\nUsage: acq_time <time> <unit>"	\
+	"\nunits: us, ns, ticks\n"
+
+#define CMD_HELP_CHANNEL			\
+	"Configure ADC channel\n"
+
+#define CMD_HELP_CH_ID				\
+	"Configure channel id\n"		\
+	"Usage: channel <channel_id>\n"
```

Comment:
I guess this should be
```suggestion
	"Usage: id <channel_id>\n"
```

Reply:
fixed